### PR TITLE
docs: improve Kafka page

### PR DESCRIPTION
**Pull request by bsmth**
__Changes:__
* Removed Postgres config section
* Clean up one-liner example for publishing JSON messages with pretty example to demonstrate the schema
* Add better instructions for making `.jar` files available to Kafka

### CODE REVIEW

**Comment by mpsq** [comment]
LGTM

File: docs/third-party-tools/kafka.md [hunk 1]
```diff
@@ -65,49 +77,114 @@ Start a Kafka server:
...
+  http://localhost:9000/exp
 ```
 
-At this point, a message can be published:
+The expected response based on the example JSON message published above will be
+the following:
 
-```shell
-bin/windows/kafka-console-producer.sh --topic example-topic --bootstrap-server localhost:9092
+```
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:120]: You need to set a language here, if you omit it, it is by default `questdb-sql`

**Review by mpsq — CHANGES_REQUESTED** [review]
minor change

File: docs/third-party-tools/kafka.md (hunk 1 above)
Comment (bsmth) [reply docs/third-party-tools/kafka.md:120]: gotcha, thanks

**Review by bluestreak01 — DISMISSED** [review]

File: docs/third-party-tools/kafka.md [hunk 2]
```diff
@@ -9,49 +9,61 @@ Support for the JDBC driver means that data can easily be exported from a Kafka
...
+match the destination QuestDB instance and a topic can be specified under the
+`topics={mytopic}` key. This example guide uses a topic `example-events` and the
+Postgres server is running on the default port `8812`.
+
+Create a file `config/connect-jdbc.properties` with the following contents:
 
-```shell
-psql -d postgres
 ```
+name=local-jdbc-sink
```
Comment (mpsq) [inline docs/third-party-tools/kafka.md:50]:
```suggestion
```shell
name=local-jdbc-sink
```

**Review by mpsq — CHANGES_REQUESTED** [review]

**Review by mpsq — APPROVED** [review]

**Pull request merged**